dev-libs/xmlrpc-c/xmlrpc-c-1.59.03.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic autotools toolchain-funcs
+
+# Upstream maintains 3 release channels: https://xmlrpc-c.sourceforge.net/release.html
+# 1. Only the "Super Stable" series is released as a tarball
+# 2. SVN tagging of releases seems spotty: https://svn.code.sf.net/p/xmlrpc-c/code/release_number/
+# Because of this, we are following the "Super Stable" release channel
+
+DESCRIPTION="A lightweight RPC library based on XML and HTTP"
+HOMEPAGE="https://xmlrpc-c.sourceforge.net/"
+SRC_URI="https://downloads.sourceforge.net/${PN}/${P}.tgz"
+
+LICENSE="BSD"
+SLOT="0/4.59"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-solaris"
+IUSE="abyss +cgi +curl +cxx +libxml2 threads test"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="test? ( abyss curl cxx )"
+
+RDEPEND="
+	dev-libs/openssl:=
+	sys-libs/ncurses:=
+	sys-libs/readline:=
+	curl? ( net-misc/curl )
+	libxml2? ( dev-libs/libxml2 )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.51.06-pkg-config-libxml2.patch
+	"${FILESDIR}"/${PN}-1.51.06-pkg-config-openssl.patch
+)
+
+pkg_setup() {
+	use curl || ewarn "Curl support disabled: No client library will be built"
+}
+
+src_prepare() {
+	default
+
+	sed -i \
+		-e "/CFLAGS_COMMON/s|-g -O3$||" \
+		-e "/CXXFLAGS_COMMON/s|-g$||" \
+		common.mk || die
+	eautoconf
+}
+
+src_configure() {
+	tc-export PKG_CONFIG
+
+	append-ldflags -Wl,-z,now
```

Comment:
I think this merits more rationale. It's already the default in 23.0 profiles.

Reply:
Ok, excluding this change.

Reply:
I have re-pushed the change without this bit.
However, when building with the default profiles, I didn’t see `-Wl,-z,now` in the logs, so apparently it doesn’t get picked up for some reason.

Reply:
The GCC is instructed to always pass it via a patch (or in the case of Clang, a configuration file). so you don't see it.